src/commands/concatenate.rs

```diff
@@ -0,0 +1,65 @@
+use asciicast::Entry;
+use failure::Error;
+use reqwest::{self, StatusCode};
+use settings::ConcatenateSettings;
+use std::fs::File;
+use std::io::copy;
+use std::io::prelude::*;
+use std::io::{self, BufReader, Write};
+use serde_json;
+use tempfile::NamedTempFile;
+
+#[derive(Debug, Fail)]
+enum ConcatenateFailure {
+    #[fail(display = "target resource not found: {}", res)]
+    NotFound { res: String },
+    #[fail(display = "something else happened (status: {})", stat)]
+    Others { stat: String },
+}
+
+pub fn go(settings: &ConcatenateSettings) -> Result<(), Error> {
```

Comment:
It looks like `cat` always just takes a file, it's `play` that can play remote or local files (plus has a bunch of other fancy things like interactive control).

Can you split this into `cat` (just the file part) and another PR for the parts of `play` you have implemented?

Reply:
`cat` can be used on both remote and local files actually.

Reply:
They don't have it in the docs buy you are correct, it does indeed work on both. Nice catch!